eZ/Bundle/EzPublishCoreBundle/Resources/config/routing/content.yml

```diff
@@ -0,0 +1,3 @@
+_ezplatform_download:
+    path: /content/download/{contentId}/{fieldIdentifier}/{downloadName}
+    defaults: {_controller: ezpublish.controller.content.download:downloadBinaryFileAction}
```

Comment:
@lolautruche but wouldn't that imply that we would still need legacy to download files?